cluster/manifests/fabric-gateway/deployment.yaml

```diff
@@ -8,9 +8,8 @@ metadata:
     application: gateway-operator
     component: fabric-gateway-controller
     version: "{{ $version }}"
-    mode: "{{ .Cluster.ConfigItems.fabric_gateway_controller_mode }}"
 spec:
-{{ if eq .Cluster.ConfigItems.fabric_gateway_controller_mode "disabled" }}
+{{ if eq .Cluster.ConfigItems.fabric_gateway_controller_enabled "false" }}
```

Comment:
minor: usually we check if it equals "true"

Reply:
done